#169520091 Add view entries endpoint

T2Wil opened the pull request:
#### What does this PR do?
- Add view entries endpoint
#### Description of Task to be completed?
- Add view entries controller
- Add database queries
- Implement tests
#### How should this be manually tested?
* have Postman
* clone the repository
* cd into directory
* run ``npm run start:dev``
* POST `api/v2/viewentries`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#169520091
#### Screenshots (if appropriate)
`INPUT VALID TOKEN`

![image](https://user-images.githubusercontent.com/30593186/68211718-c4bd2c80-ffe0-11e9-8984-0007a56d2414.png)

`ENTRIES IN DB`

![image](https://user-images.githubusercontent.com/30593186/68211761-d7cffc80-ffe0-11e9-95d7-f311846124b7.png)

T2Wil committed:
ft(db-view-entries): Add view entries endpoint
Add veiw entries controller
Add database queries
Implement tests
[Finishes 169520091]

mcaleb808 commented on server/controllers/entriesController.js:
```diff
@@ -1,115 +1,70 @@
+      userId,
+      data: entries,
     });
-  } else {
-    res.status(404).json({
```
@T2Wil what happens when a user doesn't have any entry?

T2Wil replied:
@mcaleb808
`Created a new user`
![image](https://user-images.githubusercontent.com/30593186/68214815-d1448380-ffe6-11e9-938c-18517e19ac6e.png)

`Signed in `
![image](https://user-images.githubusercontent.com/30593186/68215172-74959880-ffe7-11e9-9245-26366679d06b.png)


`Use the token to view entries`
![image](https://user-images.githubusercontent.com/30593186/68214894-f33e0600-ffe6-11e9-81c4-64bdcdd21ba3.png)

mcaleb808 replied:
@T2Wil Please also add scenario where a user is notified that he doesn't have any entry

T2Wil replied:
@mcaleb808
Added Not found status, and an error message
![image](https://user-images.githubusercontent.com/30593186/68218893-bb868c80-ffed-11e9-9129-0b03d347b8e9.png)

T2Wil committed:
ft(db-view-entries): Add view entries endpoint
Add veiw entries controller
Add database queries
Implement tests
[Finishes 169520091]
